packages/pn-personagiuridica-webapp/src/redux/notification/reducers.ts

```diff
@@ -136,29 +136,22 @@ const notificationSlice = createSlice({
         }
       }
     });
+
     builder.addCase(getReceivedNotificationPaymentUrl.rejected, (state, action) => {
       const noticeCode = action.meta.arg.paymentNotice.noticeNumber;
       const creditorTaxId = action.meta.arg.paymentNotice.fiscalCode;
-      const paymentInfo = state.paymentsData.pagoPaF24.find(
```

Comment:
Restore the old version but replace find with findIndex and paymentInfo with paymentInfoIndex. Check that paymentInfo is different from -1 and do the change:
state.paymentsData.pagoPaF24[paymentInfoIndex].pagoPa.status = PaymentStatus.FAILED;
state.paymentsData.pagoPaF24[paymentInfoIndex].pagoPa.detail = PaymentInfoDetail.GENERIC_ERROR;

Reply:
done